inc/Smartling/Helpers/MetaFieldProcessor/CloneValueFieldProcessor.php

```diff
@@ -55,9 +55,9 @@ public function processFieldPostTranslation(SubmissionEntity $submission, $field
             }
             $value = $originalValue;
         } else {
-            $this->getLogger()->warning(
+            $this->getLogger()->debug(
                 vsprintf(
-                    'Cannot clone value of metadata=\'%s\' for submission=\'%s\'. Keeping value=%s',
+                    "Metadata='%s' for submissionId='%s' not found in source metadata. Keeping value='%s'",
```

Comment:
It will be greate to use " since you touched this log. Splunk treats ' as regular character.
```suggestion
                    "Metadata=\"%s\" for submissionId=\"%s\" not found in source metadata. Keeping value=\"%s\"",
```

Reply:
Done